### ISSUE TITLE
rspec-core conflict

### ISSUE THREAD

**Issue by kleinjm** [KnapsackPro/knapsack_pro-ruby#217]
Since updating to 5.4.0, we're now getting the following warning when running `rspec` in our project.

```
The `rspec` executable in the `rspec-core` gem is being loaded, but it's also present in other gems (knapsack_pro).
If you meant to run the executable for another gem, make sure you use a project specific binstub (`bundle binstub <gem_name>`).
If you plan to use multiple conflicting executables, generate binstubs for them and disambiguate their names.
```

I believe the issue is from [this commit](https://github.com/KnapsackPro/knapsack_pro-ruby/commit/eff2d925c9d7e1e51abaccd44e15c2414c7c45c1). Would it be possible to remove the binstub from this gem please?

**Comment by 3v0k4** [comment]
Sorry @kleinjm. I'm on it.

**Comment by 3v0k4** [comment]
Fixed in v5.4.1.

Sorry for the trouble, @kleinjm 🙏

**Comment by kleinjm** [comment]
No worries at all, it was just a warning on our end. Thanks for the very fast resolution on this!
